strings_test.go

```diff
@@ -0,0 +1,50 @@
+package gitlab
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type Person struct {
+	Name string
+	Age int
+	NickNames []string
+	Address Address
+	Company *Company
+}
+
+type Address struct {
+	Street string
+	City string
+	Province string
+	Country string
+}
+
+type Company struct {
+	Name string
+	Address Address
+	Country string
+}
+
+var (
+	response       = "gitlab.Person{Name:\"name\", Age:16, NickNames:[\"n\" \"a\" \"m\" \"e\"], Address:gitlab.Address{Street:\"\", City:\"\", Province:\"\", Country:\"\"}}"
+	emptySliceResp = "gitlab.Person{Name:\"name\", Age:16, Address:gitlab.Address{Street:\"\", City:\"\", Province:\"\", Country:\"\"}}"
+)
```

Comment:
Can these be formatted for readability and maybe use backtics instead of escaping the quotes? Or does that break the tests?

Reply:
And since all of these are used only once, place declare them inside the test instead of making these globals. Thanks!

Reply:
* backticks will break the tests. So not changing anything there.
* declared them inside test.